### ISSUE TITLE
[DOC] Update projects page

### ISSUE THREAD

**Issue by TonyBagnall** [aeon-toolkit/aeon#2122]
### Describe the issue linked to the documentation

the project page needs updating and we should include a list of completed projects.

**Comment by adityagh006** [comment]
Hi @TonyBagnall, I’d like to work on this issue! Could you clarify where the projects page is located? Also, should the completed projects list include just titles, or should we add descriptions and links to relevant PRs/issues?

Looking forward to contributing!

**Comment by MatthewMiddlehurst** [comment]
I dont think this is a good issue for first time contributors. If you have project ideas feel free to suggest them, but i would not go through a large amount of effort to write them up at first.

**Comment by adityagh006** [comment]
Got it, thanks for the heads-up! I’ll hold off on making major changes for now. If I come across any project ideas that might fit, I’ll be sure to suggest them. Let me know if there’s anything else I can help with!

**Comment by Dronten** [comment]
is this issue solved or can i work on it ?

**Comment by TonyBagnall** [comment]
this issue is old and too generic. We should update the page but also assign people to do so
